### DISCUSSION TITLE
Request: Media controls under current audio device

### DISCUSSION THREAD

**Discussion by spikespaz** [File-New-Project/EarTrumpet#670] [Ideas]
An example of what I'm looking for.

![Screenshot Photoshopped](https://user-images.githubusercontent.com/12502988/46319063-878c9c80-c58d-11e8-86c1-2b30379dc997.png)

**Comment by DragoCubed** [comment]
I would also like the title + artist

**Comment by spikespaz** [comment]
@DragoCubed I like the idea but I would like the option to turn it off if I don't like it. Just throwing that out there, both of these features should be optional.

**Comment by nta** [comment]
Even _Windows_ is getting this functionality now, EarTrumpet is literally worse than the base volume controls by then.

Please do this before 19H1 ships 👍

**Comment by riverar** [comment]
Is there an indication Windows 19H1 is getting this?

**Comment by nta** [comment]
Given that MSFT pushes back features all the time, of course not. It's in the bits, though, as `EnablePlaybackControlsUX`.

**Comment by ShankarBUS** [comment]
#475
